app/imports/ui/components/administrator/ViewTeams.jsx

```diff
@@ -104,64 +90,60 @@ const ViewTeams = ({ participants, teams, teamChallenges, teamParticipants }) =>
   };
 
   return (
-      <Grid container centered>
-        <Grid.Row>
-          <Grid.Column width={16}>
-            <div style={{
-              backgroundColor: '#E5F0FE', padding: '1rem 0rem', margin: '2rem 0rem',
-              borderRadius: '2rem',
-            }}>
-              <Header as="h2" textAlign="center">View Teams ({filteredTeams.length})</Header>
-            </div>
-          </Grid.Column>
-        </Grid.Row>
-        <Grid.Row>
-          <Button onClick={handleDownload}>Download Team Captain emails</Button>
-        </Grid.Row>
-        <Grid.Row>
-          <Grid.Column width={4}>
-            <Segment style={stickyStyle}>
-              <Form>
-                <Form.Field>Select a filter</Form.Field>
-                {/* <Form.Field> */}
-                {/*  <Checkbox checked={filterValue === 'Challenge'} label="No challenge" onChange={handleChange} */}
-                {/*            radio name='checkboxRadioGroup' value='Challenge' */}
-                {/*  /> */}
-                {/* </Form.Field> */}
-                <Form.Field>
-                  <Checkbox checked={filterValue === 'NonCompliant'} label="Non Compliant" onChange={handleChange}
-                            radio name='checkboxRadioGroup' value='NonCompliant'
-                  />
-                </Form.Field>
-                <Form.Field>
-                  <Checkbox checked={filterValue === 'NoDevPost'} label="No devpost" onChange={handleChange}
-                            radio name='checkboxRadioGroup' value='NoDevPost'
-                  />
-                </Form.Field>
-                <Form.Field>
-                  <Checkbox checked={filterValue === 'NoGitHub'} label="No GitHub" onChange={handleChange}
-                            radio name='checkboxRadioGroup' value='NoGitHub'
-                  />
-                </Form.Field>
-                <Form.Field>
-                  <Checkbox checked={filterValue === 'None'} label="None" onChange={handleChange}
-                            radio name='checkboxRadioGroup' value='None' />
-                </Form.Field>
-              </Form>
-            </Segment>
-          </Grid.Column>
-          <Grid.Column width={12}>
-            <Item.Group divided>
+      <Container className="mt-5 mb-5">
+        <Row className="mb-4">
+          <Col>
+            <Card bg="light" className="text-center p-3 rounded">
+              <Card.Title>View Teams ({filteredTeams.length})</Card.Title>
+            </Card>
+          </Col>
+        </Row>
+        <Row className="mb-4">
+          <Button
+              variant="primary"
+              onClick={handleDownload}
+              id={COMPONENT_IDS.DOWNLOAD_TEAM_CAPTAIN_EMAIL}
```

Comment:
id is missing an s. Should be COMPONENT_IDS.DOWNLOAD_TEAM_CAPTAIN_EMAILS. app/imports/ui/components/administrator/ViewTeams.jsx 
Line 105